ftl/cached/ftl_node_cached_yaml.py

```diff
@@ -0,0 +1,72 @@
+"""A script to generate a cloudbuild yaml."""
+
+import os
+import yaml
+
+_TEST_DIR = '/workspace/ftl/node/testdata'
+_NODE_BASE = 'gcr.io/google-appengine/nodejs:latest'
+
+
+def main():
+    app_dir = 'packages_test'
+    path = 'gcr.io/ftl-node-test/node/cache/%s' % app_dir
+    name = path + ':latest'
+
+    cloudbuild_yaml = {
+        'steps': [
+            # We need to chmod in some cases for permissions.
+            {
+                'name': 'ubuntu',
+                'args': ['chmod', 'a+rx', '-R', '/workspace']
+            },
+            # Build the node builder par file
+            {
+                'name': 'gcr.io/cloud-builders/bazel',
+                'args': ['build', 'ftl:node_builder.par']
+            },
+            # Build the node builder par file
+            {
+                'name':
+                'gcr.io/cloud-builders/bazel',
+                'args': [
+                    'run', '//ftl/node/cached:node_cached', '--', '--base',
+                    _NODE_BASE, '--name', name, '--directory',
+                    os.path.join(_TEST_DIR + app_dir)
+                ]
+            },
+            # Get image digests so that we can delete uploaded images
+            {
+                'name':
+                'gcr.io/cloud-builders/gcloud',
+                'entrypoint':
+                '/bin/bash',
+                'args': [
+                    '-c', 'gcloud container images list-tags ' + path + ' | ' +
```

Comment:
Can you do this part in the test itself?

Reply:
done